### ISSUE TITLE
HW_III Rubric Inconsistent with Assignment

### ISSUE THREAD

**Issue by CooperStansbury** [betteridiot/b575f19#9]
Hi @betteridiot @mitreac 

I noticed that this rubric item below is not mentioned all at in the assignment. Is this on purpose?

![image](https://user-images.githubusercontent.com/35812893/66225838-734f1400-e6a7-11e9-90e6-0233360f6331.png)

Thanks,

**Comment by betteridiot** [comment]
You are correct. It must have been removed from the instructions during some of the revisions. This has been reflected on the revised rubric

![image](https://user-images.githubusercontent.com/25447319/66227626-8cf25a80-e6ab-11e9-9385-d8d5442ef621.png)

**Comment by betteridiot** [comment]
Thank you
